index.js

```diff
@@ -21,6 +21,7 @@ const depsWriter = path.join(closureBinPath, 'depswriter.py');
 /**
  * Run the Closure Compiler with the provided options.
  * @param {Object} options The compiler options.
+ * @param {boolean=} opt_useNative
```

Comment:
Is this from a previous iteration? It’s not included in the function params, nor used.

Reply:
Yup, fixed.